test/extended/etcd/vertical_scaling.go

```diff
@@ -112,4 +112,78 @@ var _ = g.Describe("[sig-etcd][Serial] etcd [apigroup:config.openshift.io]", fun
 		err = errors.Wrap(err, "scale-down: timed out waiting for only 3 Running master machines")
 		o.Expect(err).ToNot(o.HaveOccurred())
 	})
+
+	// The following test covers replacing a failed master machine from 3 master nodes cluster.
+	// It starts by setting one of the master machine to be failed/unhealthy.
+	// The failed machine is expected to be replaced automatically.
```

Comment:
User's must trigger this though

Reply:
yes, thats why I make this API call 

```
                 err = machineClient.Delete(ctx, machineName, metav1.DeleteOptions{})
		o.Expect(err).ToNot(o.HaveOccurred())
		framework.Logf("successfully deleted the machine %q from the API", machineName)
```

Reply:
Ok cool, may just want to update this comment

Reply:
sure, i am just trying to get success from `make verify` :D

Reply:
> 

updated :)